docs/manual/introduction/Browser-support.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<base href="../../" />
+	<script src="list.js"></script>
+	<script src="page.js"></script>
+	<link type="text/css" rel="stylesheet" href="page.css" />
+</head>
+<body>
+	<h1>[name]</h1>
+
+	<h2>Overview</h2>
+	<div>
+		<p>
+			<strong>TL;DR</strong> three.js supports most modern browsers, including Internet Explorer 11 and above.</p>
```

Comment:
I think you can leave out the "TL;DR". 

I think you can just say "Three.js supports all modern browsers" and leave it at that.

Reply:
Actually I would word this a bit differently - the introduction doesn't make it that clear what this document is for. 

Something like:

"Three.js can use WebGL to render you scenes on all modern browsers. For older browsers, especially Internet Explorer version 10 and below, you may have to fallback to one of the other renderers (CSS2DRenderer, CSS3DRenderer, SVGRenderer,  CanvasRenderer). Additionally, you may have include some polyfills, especially if you are using files from the /examples folder." 

Note: if you don't need to support these old browsers, then it is not recommended to use the other renderers as they are slower and support less features than the WebGLRenderer. 

## Browsers that support WebGL:
Google Chrome 9+, Firefox 4+, Opera 15+, Safari 5.1+, Internet Explorer 11 and Microsoft Edge (link to caniuse)